protocol/messenger_handler.go

```diff
@@ -351,6 +352,7 @@ func (m *Messenger) createContactRequestNotification(contact *Contact, messageSt
 		Author:    messageState.CurrentMessageState.Contact.ID,
 		Timestamp: messageState.CurrentMessageState.WhisperTimestamp,
 		ChatID:    contact.ID,
+		Read:      contactRequest.ContactRequestState == common.ContactRequestStateAccepted || contactRequest.ContactRequestState == common.ContactRequestStateDismissed,
```

Comment:
>We have Accepted and Dismissed falgs in notification object. But those field are not used (yet). My suggestion is to fill them in same place i'm setting read flag in this PR

I think so :+1: For instance, a couple lines above and we see the notification might be dismissed in the DB, but then here we build a new struct instance that doesn't have the `Dismissed` field. To be consistent with what's in the DB, I think we should also define the other fields.

Reply:
Working on it. And one thing is not clear for me:

When we accept or decline contact request we should mark linked notification as Read and Accepted?

Or for accepting contact request we should mark notification as Read and Accepted and for declining contact request we should mark notification as Read and Dismissed?

For me second way is closer, but we have filtering based on Dismissed flag in activity_center_persistance

Reply:
Just for the record, @MishkaRogachev and I discussed outside Github what exactly are these dismissed/accepted fields, because up until this point, both Desktop and Mobile apps relied exclusively on the notification states. For example, to know if a contact request has been `accepted`, clients use the `contactRequestState` in the RPC response. https://github.com/status-im/status-go/blob/develop/protocol/common/message.go#L58

Reply:
I've added dismissed and accepted with latest commit